api/views/v1/generatedTasks.py

```diff
@@ -60,38 +60,34 @@ def generateTasks(projectData, authorizationToken):
         # Check
         serializer = GeneratedTaskSerializer(data=projectData)
         if not serializer.is_valid():
-            return serializer.errors, status.HTTP_400_BAD_REQUEST
+            return {"message":serializer.errors}, status.HTTP_400_BAD_REQUEST
 
         projectName = serializer.validated_data["name"]
-        projectDescription = serializer.validated_data["description"]
         projectID = serializer.validated_data["projectID"]
         totalProjectMinutes = 0
 
         # Load ChatGPT creds
         load_dotenv()
         openai.api_key = os.getenv("CHATGPT_API_KEY")
 
-        print("========================")
-        print(f"{projectName=}\n{projectDescription=}")
-
         # Call ChatGPT
         completion = openai.chat.completions.create(
             model="gpt-3.5-turbo",
             messages=[
                 {
                     "role": "system",
                     "content": """You are an assistant for quayside.app, a project management team. 
-                    You are given, as input, infromation regarding a project that a single 
-                    person or a team wants to take on. 
-                    Break the project into less than 5 tasks and less than 5 subtasks for each task,
-                    and list them hierarchically in the format where task 1 has subtasks 1.1, 1.2,...
-                    and task 2 has subtasks 2.1, 2.2, 2.3,... and so forth. Make sure that every 
-                    task is on one line after the number. For each subtask without it's own hierarcy, 
-                    provide a time estimation in minutes in square brackets with the label "minutes". 
+                    You are given as input a project or task that a single person or a team 
+                    wants to take on. Divide the task into less than 5 subtasks and list them 
+                    hierarchically in the format where task 1 has subtasks 1.1, 1.2,...
+                    and task 2 has subtasks 2.1, 2.2, 2.3,... and so forth and allow for subtasks to 
+                    have their own hierarcy in the format of 1.1.1, 1.1.2, 1.13,... and so forth. For each subtask without it's own hierarcy, 
+                    provide a time estimation in minutes in square brackets with the label "minutes". Do not give a minute range.
+                    Make sure that every task is on one line after the number and has a time estimation. 
                     NEVER create new paragraphs within a task or subtask.
                     """,
                 },
-                {"role": "user", "content": f"Project Name: {projectName}\nProject Description: {projectDescription}"},
```

Comment:
This line needs to be reverted back to this so the description is actually affecting the ChatGPT output.